config.example.yaml

```diff
@@ -66,11 +78,19 @@ git:
 
 groups:
   - name: golang
+    apply-on-sub-projects: false
     checks:
       - go-version-latest
       - go-main
     when:
       - golang-projects
+  - name: paseto-group
+    apply-on-sub-projects: true
+    checks:
+      - paseto-migration-check
+    when:
+      - ruby-controller
+      - ruby-projects
```

Comment:
a rule like ruby-projects should be executed only on the project level and assume satisfied for all the related subprojects